### ISSUE TITLE
Error Code 4:kOPT values for profile 0 violate shape constraints

### ISSUE THREAD

**Issue by xaiocaibi** [NVIDIA/TensorRT#2472]
## Description
When I converted the onnx model to the trt model，i got this error：

[11/10/2022-17:35:34] [TRT] [E] 4: [shapeCompiler.cpp::evaluateShapeChecks::1180] Error Code 4: Internal Error (kOPT values for profile 0 violate shape constraints: node_of_143: ISliceLayer has out of bounds access on axis 1 Out of bounds access for slice.)
[11/10/2022-17:35:34] [TRT] [E] 2: [builder.cpp::buildSerializedNetwork::751] Error Code 2: Internal Error (Assertion engine != nullptr failed. )

this is my node_of_143 ：
![image](https://user-images.githubusercontent.com/18330731/201084496-45a15190-4e9f-4092-9e61-14b3f60b079f.png)

## Environment

**TensorRT Version**: TensorRT-8.5.1.7
**NVIDIA GPU**: T4
**NVIDIA Driver Version**: TensorRT-8.5.1.7
**CUDA Version**: release 11.8
**CUDNN Version**: 8.6
**Operating System**: centos 8
**Python Version (if applicable)**: 3.7
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.3
**Baremetal or Container (if so, version)**: 

 

I hope I can get your help. 3Q

**Comment by zerollzeng** [comment]
Looks like the `indices` you are using is out of bounds for the dynamic shape you set.

**Comment by xaiocaibi** [comment]
已收到                                                                                                            蔡成林

**Comment by nvpohanh** [comment]
Please provide the ONNX file so that we can investigate. Thanks

**Comment by xaiocaibi** [comment]
已收到                                                                                                            蔡成林

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks. thanks all!

**Comment by xaiocaibi** [comment]
已收到                                                                                                            蔡成林
